tests/launch-listener-test-util-libs/test-launch-listener-03/spotbugs-exclude.xml

```diff
@@ -0,0 +1,21 @@
+<!--
+  ~  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Do we need this empty spotbugs-exclude files

Reply:
no need. removed.